### ISSUE TITLE
TwigExceptionController

### ISSUE THREAD

**Issue by peshi** [FriendsOfSymfony/FOSRestBundle#1372]
Due to recent exception wrapper changes, this emerged.
```
ContextErrorException in TwigExceptionController.php line 23:
Runtime Notice: Declaration of FOS\RestBundle\Controller\TwigExceptionController::createView() should be compatible with FOS\RestBundle\Controller\ExceptionController::createView(Exception $exception, $code, array $templateData, Symfony\Component\HttpFoundation\Request $request, $showException)
```

**Comment by lsmith77** [comment]
I assume this is on 1.8 beta1?

**Comment by Ener-Getick** [comment]
No @lsmith77 this is on 2.0.0-BETA1

**Comment by shakly** [comment]
If it can help you, i have the same issue when i get a NotFoundHttpException or MethodNotAllowedHttpException

**Comment by Ener-Getick** [comment]
Fix in #1374 thank you !
